backend/tests/unit/api/Controllers/Admin/AccessRequestControllerTest.cs

```diff
@@ -40,17 +41,18 @@ public void GetAccessRequests_Success()
             var accessRequest2 = EntityHelper.CreateAccessRequest(2);
             var accessRequests = new[] { accessRequest1, accessRequest2 };
             var paged = new Entity.Models.Paged<Entity.AccessRequest>(accessRequests);
-            service.Setup(m => m.User.GetAccessRequests(It.IsAny<int>(), It.IsAny<int>(), It.IsAny<string>(), It.IsAny<Entity.AccessRequestStatus?>())).Returns(paged);
+
+            service.Setup(m => m.User.GetAccessRequests(It.IsAny<AccessRequestFilter>())).Returns(paged);
 
             // Act
-            var result = controller.GetPage(1, 10);
+            var result = controller.GetPage(1, 10, null, null, null, null, Entity.AccessRequestStatus.OnHold);
```

Comment:
You don't need to change this now, but here you can see the result of your choice to add the query parameters as arguments on the endpoint.  If you look at other implementations with filters you can see a cleaner approach where it isn't required.